src/lib.rs

```diff
@@ -352,11 +413,11 @@ where
         self.write_many(Registers::DioMapping1, &mut val)
     }
 
-    fn reset_fifo(&mut self) -> Result<(), Ecs, Espi> {
+    pub fn reset_fifo(&mut self) -> Result<(), Ecs, Espi> {
         self.write(Registers::IrqFlags2, 0x10)
     }
 
-    fn wait_mode_ready(&mut self) -> Result<(), Ecs, Espi> {
+    pub fn wait_mode_ready(&mut self) -> Result<(), Ecs, Espi> {
```

Comment:
What is the reason to make this public?

Reply:
That in particular could be private again.

Reply:
Actually I'd like to leave that public so I can use it later in my projects.